app/src/NativeBase/Components/List.tsx

```diff
@@ -116,18 +130,65 @@ const List: FC<ListProps> = ({ data, mode, options, searchScreen, type }) => {
       return (
         <FlatList
           data={data as Projects}
-          keyExtractor={project => project.res_id}
           renderItem={({ item }) => {
             return (
-              <ProjectListItem
-                onPress={() => {
-                  navigate('ProjectDetail', { item, key: item.res_id })
-                }}
-              >
-                <ProjectSummary project={item} />
-              </ProjectListItem>
+              <Card margin="2">
```

Comment:
I'm finding it slightly hard to work out from the Figma design how much vertical padding there should be inside the card, but feels like this maybe needs `paddingY="2"`?